src/main/resources/application.yml

```diff
@@ -56,3 +59,16 @@ spring:
     static-locations: classpath:/api/
   config:
     import: optional:file:config/local-properties.yml
+
+otel:
+  sdk:
+    disabled: false # set to true to disable all open telemetry features
+  traces:
+    sampler:
+      probability: ${SAMPLING_PROBABILITY:0}
```

Comment:
Linking @dvoet 's correction in the ECM upgrade -- I think sampling config is handled differently.

https://github.com/DataBiosphere/terra-external-credentials-manager/pull/161#discussion_r1426768904

Reply:
Thanks!